### ISSUE TITLE
Can't hide the navigation bar

### ISSUE THREAD

**Issue by htttbme** [toneiv/UbikiTouch#84]
I have already upgraded to Android 11, then I found that I can't hide the navigation bar anymore. Is that the problem with Android 11 or I need to add some more steps to hide?

**Comment by toneiv** [comment]
Unfortunately the possibility to hide the navigation bar via ADB has been removed by Google in Android11.
I am aware of the problem and am looking for alternative solutions. 
I will update the application quickly if possible.

**Comment by Veiranx** [comment]
> 
> 
> Unfortunately the possibility to hide the navigation bar via ADB has been removed by Google in Android11.
> I am aware of the problem and am looking for alternative solutions.
> I will update the application quickly if possible.

Thank you; we're hoping you and the community can come up with some good ideas!

**Comment by toneiv** [comment]
Thank you. I need to find an Android 12 device with MIUI to replicate this. 
If it's successful, I wouldn't hesitate to include the feature in a future update.

**Comment by cableghost** [comment]
Is hiding the navbar on Android 11 still not possible on stock build?

**Comment by toneiv** [comment]
Hi. Still not as far as I know...

**Comment by Sparksys98** [comment]
@toneiv Is this implemented for later Android versions?

**Comment by toneiv** [comment]
No. The methods that worked in versions prior to Android 11 have been removed and are no longer available.

**Comment by Marcosaurios** [comment]
Asking to just confirm it, I guess this is still not possible on Android 14 right? I tried with the great ADB installer you made (https://ubikitouch.toneiv.eu/adb/#/) but I get the following error: `Permission WRITE_SECURE_SETTINGS is currently not granted for UbikiTouch`

**Comment by toneiv** [comment]
I've just taken the time to check again: Google hasn't reactivated this option in Android 14. As far as I know, there is currently no ADB option.

**Comment by toneiv** [comment]
@Nomis9 I didn't know that, thanks for the info. I'll see if it's possible to add this option for MIUI devices.

**Comment by SyedArbaazHussain** [comment]
> `adb shell cmd settings put global force_fsg_nav_bar 1` works for me on Android 12 and MIUI

Running this command on Android 15 with HyperOS 2 for Poco on my Poco F7 works.

@toneiv Add this in the docs and the app for hyperos devices at least and if possible grantable by acquiring Shizuku privileges.

**Comment by rivai45** [comment]
Maybe a manual toggle via a Quick Settings tile could be a practical workaround here, instead of full automation.
Android tends to reset immersive mode anyway, so a quick way to reapply it might be enough for some users.

I tried building a small app with auto-start + QS tile for this, and it works decently as a manual solution.

*tested on Poco F6 and F7 with HyperOS 3
